Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
   stages {
```

Comment:
I'm wondering if it makes sense at this point to add parallelization for these tests. We use this in other projects, for example here: https://github.com/conjurdemos/kubernetes-conjur-demo/blob/master/Jenkinsfile#L13

Reply:
Excellent point, I'll add it, thanks!

Reply:
Unfortunately it's not that simple. I created an issue for this, see #231.

Reply:
Ok. It's fine if that gets added later, but for now there is still a failing build on this one.